examples/grafana-agent-auto-instrumentation/ebpf-otel/README.md

```diff
@@ -0,0 +1,67 @@
+# OpenTelemetry eBPF Profiler Example
+
+**⚠️ Important: Linux-only Support**
+This example can only be run on Linux systems (amd64/arm64) as it relies on eBPF technology which is specific to the Linux kernel. The profiler requires privileged access to system resources.
+For more details refer to the OpenTelemetry ebpf profiler [docs](https://github.com/open-telemetry/opentelemetry-ebpf-profiler).
+
+These examples demonstrates:
+1. OpenTelemetry eBPF profiler collecting system-wide profiles
+2. OpenTelemetry Collector receiving and processing the data from the profiler
+3. Pyroscope receiving and visualizing the profiles via Grafana
+
+## Prerequisites
+**⚠️ Important:** Since the [profiler image](https://hub.docker.com/r/otel/opentelemetry-ebpf-profiler-dev) is not publicly available yet, you need to build the profiler binary first:
```

Comment:
```suggestion
**⚠️ Important:** Since the [profiler image](https://hub.docker.com/r/otel/opentelemetry-ebpf-profiler-dev) is not publicly available yet, you need to build the profiler binary first.
```